grails-app/controllers/au/org/ala/spatial/service/LogController.groovy

```diff
@@ -60,18 +52,15 @@ class LogController {
      *
      * @return
      */
+    @RequireLogin
```

Comment:
This is at class level. Remove duplicate method level annotation.

Reply:
Done